tests/src/core/testqgsvectorlayer.cpp

```diff
@@ -334,7 +334,7 @@ void TestQgsVectorLayer::testAddTopologicalPoints()
   QVERIFY( layerLine->isValid() );
```

Comment:
Can you also add tests for -1 and 1 return code?

Reply:
Sure, added in https://github.com/qgis/QGIS/pull/43854/commits/92231d74c861f6d40997efbe1e3a4a6de8f849fa